app/src/main/java/org/mozilla/fenix/customtabs/CustomTabToolbarMenu.kt

```diff
@@ -46,7 +47,8 @@ class CustomTabToolbarMenu(
     override val menuBuilder by lazy { BrowserMenuBuilder(menuItems) }
 
     /** Gets the current custom tab session */
-    private val session: TabSessionState? get() = sessionId?.let { store.state.findTab(it) }
+    @VisibleForTesting
+    internal val session: CustomTabSessionState? get() = sessionId?.let { store.state.findCustomTab(it) }
```

Comment:
Ah good catch! This should really have been `findTabOrCustomTab` here as in other places, and never just `findTab`. However, just restricting to custom tabs here is fine too as this menu isn't used anywhere else.